README.md

```diff
@@ -2,40 +2,55 @@
 Let's make a planetary system!
 
 ## Learning Goals
-### Wave 1
-- Create a custom **class**
-- Create and use an **initialize** method in your class
-- Create and use **instance variables** in your class
 
-### Wave 2
-- Use an **array** to store a list of objects
-- Create and use a **method**
+- Practice creating Custom classes
+- Use a hash to initialize an object
+- Create an object which contains a collection of other objects
+- Practice creating instance methods
+
 
 ## Baseline
```

Comment:
Yes - I think the baseline requirements should have the same level of complexity at the original ones, only creating and instantiating a `SolarSystem` class with a name (or some other "flat" object)